packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -168,6 +169,12 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public fixedHeader?: string;
 
+	/**
+	 * YPosition of the searchFilter box for the table and the column
+	 */
+	@Input()
+	public searchFilterYPosition?: "above" | "below" = "below";
```

Comment:
This should not be a separate input on the component, but instead it should be put together with all the properties of the table filter here: [/packages/stark-ui/src/modules/table/entities/table-filter.intf.ts](https://github.com/NationalBankBelgium/stark/blob/master/packages/stark-ui/src/modules/table/entities/table-filter.intf.ts)